cli/cmd/agent.go

```diff
@@ -158,6 +161,77 @@ To list all active agents in your environment.
 NOTE: New agents could take up to an hour to report back to the platform.`,
 		RunE: installRemoteAgent,
 	}
+
+	agentCTFCmd = &cobra.Command{
+		Use:   "ctf",
+		Short: "Install the datacollector agent on all remote hosts",
+	}
+
+	agentCTFAWSCmd = &cobra.Command{
+		Use:   "aws",
+		Short: "Install the datacollector agent on all remote AWS hosts",
+	}
+
+	agentCTFAWSEC2ICCmd = &cobra.Command{
+		Use:   "ec2ic",
+		Short: "Use EC2InstanceConnect to install the datacollector agent on all remote AWS hosts",
+		RunE:  awsCaptureTheFlagEC2IC,
+		Long: `This command installs the agent on all remote AWS EC2 hosts in your account
+using EC2InstanceConnect.
+
+To only install on instances in a specified list of regions:
+
+	lacework agent ctf aws ec2ic --include_regions us-west-2 us-east-2
```

Comment:
When mentioning an example command use spaces, not tabs.

```suggestion
    lacework agent ctf aws ec2ic --include_regions us-west-2 us-east-2
```

4 spaces to be explicit so that this translates into Markdown like this https://docs.lacework.com/cli/commands/lacework_agent_install/